Source/Blazorise/Components/FocusTrap/FocusTrap.razor.cs

```diff
@@ -38,26 +38,45 @@ protected override async Task OnAfterRenderAsync( bool firstRender )
             }
         }
 
-        internal async Task SetFocus()
+        /// <summary>
+        /// Sets the focus trap.
+        /// </summary>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public async Task SetFocus()
```

Comment:
So there's a `SetFocus` so we're trapper inside a container for tabbing/focus right? What if I want to get out of the "trap"?

Reply:
Then `Active` would be set to `false`.

Reply:
So `Active` + `SetFocus` does nothing
`Active` + `SetFocus` sets the trap, and if I want to disable after, I just set `Active` to false, that's it, right?

Reply:
`Active` will basically enable tabindex in the wrapper components. But it will not do anything to actually focus the input inside of its content. With `SetFocus` we force it to focus on the wrapper which then keeps track of everything else, tab press, leave event, etc.